### ISSUE TITLE
Discuss <scriptNote> and how to use in metadata context

### ISSUE THREAD

**Issue by riedde** [music-encoding/metadata-ig#15]
discussion from Slack-Channel (Feb, 8th 2022):
@irmlindcapelle  [14:15 Uhr UTC+1]
Perhaps a new element for the header: In some cases it is necessary to describe the notation (CMN, mensural, tablature etc.) used for the notated music but until now there is no element. We have `<scriptNote>` but this is fixed on text. Could it be opened for music?

@margrethesb [14:23 Uhr]
Irmlind: that sounds like a very good idea!

@irmlindcapelle [14:41 Uhr]
Some reflections of Johannes on this (in the body of MEI the information is possible as attribute to staffDef): Ich empfinde die Aufteilung in Header und Musik zunehmend als unnötige Einschränkung – nicht im Gebrauch, sondern im Denken. Es ist aus meiner Sicht völlig legitim, auch “Metadaten” im Body abzulegen und andersrum. Es müsste aus meiner Sicht also nicht zwingend nochmal ein separates Feld im Header gesucht werden – wenn denn die bisherige Lösung hinreichend (klar) ist. Da bräuchte es vermutlich mindestens noch die Erweiterung auf `<scoreDef>` (um nicht zwingend die Besetzung angeben zu müssen), aber dann könnte es evtl. schon gehen. Schwierig ist es, wenn wir da zunehmend Doppelstrukturen aufbauen, die ja auch immer dokumentiert und später in den Instanzen koordiniert werden müssen…

@riedde [17:28 Uhr]
I will open an issue in our repo! It's worth to discuss this.

**Comment by KristinaRichts** [comment]
> Perhaps a new element for the header: In some cases it is necessary to describe the notation (CMN, mensural, tablature etc.) used for the notated music but until now there is no element. We have <scriptNote> but this is fixed on text. Could it be opened for music?

Could this perhaps be solved via the @type attribute on the `mdiv`? I had a similar problem with AV media encoding, there it would most likely be solved via @mimetype at `avFile` in `body`.

**Comment by ahankinson** [comment]
I don't see any restriction on `<scriptNote>` or `<scriptDesc>` being used only for text; Is there a reason for the assertion that its use is restricted to textual content only?

**Comment by KristinaRichts** [comment]
That's right, @ahankinson , neither do I. My suggestion was in the direction of a classification when it is a matter of basic classification of the type of notation and no further description is required. A more detailed description could then be captured within `scriptDesc` or `scriptNote`. Right? As far as I see there is no element for the notation type yet.

**Comment by ahankinson** [comment]
There is the `att.notationType` attributes: https://music-encoding.org/guidelines/dev/attribute-classes/att.notationType.html

These accept values from the `data.NOTATIONTYPE` data type: https://music-encoding.org/guidelines/dev/data-types/data.NOTATIONTYPE.html

**Comment by KristinaRichts** [comment]
Ah, ok, thank you, that's good. But where do you put this information if you only capture the metadata?

**Comment by bwbohl** [comment]
Maybe the notion of using them for text exclusively is due to:

1. The description of `scriptDesc` explicitly mentioning `letters` and `characters`
   >Contains a description of the letters or characters used in an autographic item
2. the fact that the descriptions state that the elements are based on the corresponding TEI elements
